hudi-common/src/main/java/org/apache/hudi/common/table/view/AbstractTableFileSystemView.java

```diff
@@ -436,30 +431,7 @@ private List<StoragePathInfo> getAllFilesInPartition(String relativePartitionPat
    * @param partition partition to be loaded if not present
    */
   protected void ensurePartitionLoadedCorrectly(String partition) {
-
-    ValidationUtils.checkArgument(!isClosed(), "View is already closed");
```

Comment:
Let's keep as it is, the metadata fetch and the debug msg are somehow diffent, one thing we can do it moving these two methods together.

Reply:
Added it back in the last commit